man/IDateTime.Rd

```diff
@@ -88,11 +88,11 @@ second(x)
 minute(x)
 hour(x)
 yday(x)
-wday(x)
+wday(x, local_name = FALSE)
 mday(x)
 week(x)
 isoweek(x)
-month(x)
+month(x, local_name = FALSE)
 quarter(x)
 year(x)
 yearmon(x)
```

Comment:
I have never used these helper functions, because I think strftime is sufficient. (do we need to add this functionality? it seems like it would less maintenance for us, if we just encouraged strftime/as.Date with format codes, which seems more standard anyway)

Also it seems strange to me that local_name can only be provided for month and wday, and not the others (minute, hour, etc).
what is the justification? because https://github.com/Rdatatable/data.table/issues/1286 only asked for those two?

Reply:
Yea, I'm also not totally convinced. The advantage of these functions is they return an integer-valued answer, whereas `strftime()` is returning string. Re-reading the issue+comments, I see two reasons to prefer supporting this:

 1. These flexibly take string or date as input -- no need to check if `as.IDate()` is required first
 2. We could make life a lot easier by generating an factor [ref](https://github.com/Rdatatable/data.table/issues/1286#issuecomment-309570372). It does feel a little out-of-scope, though -- months is easy enough, but whether to start the week on Sunday or Monday is also locale-dependent. #6025 gets at the same issue -- do we really want a 3rd parameter here for a simple function, though?

WDYT @jangorecki, maybe we just close this FR/PR?